src/Network/AWS/S3/StreamingUpload.hs

```diff
@@ -121,21 +122,21 @@ streamUpload mChunkSize multiPartUploadDesc =
 
     logStr :: String -> m ()
     logStr msg  = do
-      logger <- view envLogger
+      let logger = _envLogger env
       liftIO $ logger Debug $ stringUtf8 msg
 
     startUpload :: ConduitT (Int, S) Void m
                     (Either (AbortMultipartUploadResponse, SomeException)
                     CompleteMultipartUploadResponse)
     startUpload = do
-      multiPartUpload <- lift $ send multiPartUploadDesc
-      when (multiPartUpload ^. cmursResponseStatus /= 200) $
+      multiPartUpload <- lift $ send env multiPartUploadDesc
+      when (multiPartUpload ^. createMultipartUploadResponse_httpStatus /= 200) $
         throwM $ UnableToCreateMultipartUpload multiPartUpload -- "Failed to create upload"
       lift $ logStr "\n**** Created upload\n"
 
-      let Just upId = multiPartUpload ^. cmursUploadId
-          bucket    = multiPartUploadDesc  ^. cmuBucket
-          key       = multiPartUploadDesc  ^. cmuKey
+      let Just upId = multiPartUpload ^. createMultipartUploadResponse_uploadId
+          bucket    = multiPartUploadDesc  ^. createMultipartUpload_bucket
+          key       = multiPartUploadDesc  ^. createMultipartUpload_key
```

Comment:
We're recommending people use `generic-lens` or `optics` to lens into the records. Maybe you want to do this here?